src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransactionRepository.cs

```diff
@@ -165,7 +165,7 @@ public TransactionRepository(LevyDeclarationProviderConfiguration configuration,
                 }
             }
 
-            return transactionDownloadLines.ToList();
+            return transactionDownloadLines.OrderByDescending(txn => txn.DateCreated).ToList();
```

Comment:
Could this be done in the `GetAllTransactionDetailsForAccountByDate` sproc instead? Not an issue if not, your call.

Reply:
Yeah, did consider that but I don't think the sproc could know in advance what the required order should be. Also tend to favour getting the date out of SQL as fast as possible and leave presentation stuff to the consumer.